content/localdotcom/drone-azure-pipelines/index.md

```diff
@@ -0,0 +1,116 @@
+---
+version: '1.0.0'
+date: 2020-01-20T00:00:00+00:00
+title: Azure Pipelines
+author: localdotcom
+tags: [ infrastructure, ops, azure]
+logo: azure_pipelines.svg
+repo: localdotcom/drone-azure-pipelines
+image: dclocal/drone-azure-pipelines:1.0.0
+---
+
+This plugin can be used to bulk run Azure pipelines. The below sample pipeline configuration demonstrates simple usage:
+
+```yaml
+kind: pipeline
+name: Bulk run Azure pipelines
+
+platform:
+  os: linux
+  arch: amd64
+
+steps:
+- name: Run build pipeline
+  image: dclocal/drone-azure-pipelines:1.0.0
+  settings:
+    action: queue_build
+    definitions: build_definition_id

```

Comment:
Slice and string in one doesn't make sense

Reply:
Extra slice removed. My mistake.